docs/samples/Microsoft.ML.Samples/Static/FeatureSelectionTransform.cs

```diff
@@ -0,0 +1,122 @@
+            // directly, but it needs to be trained on data using .Fit(), and it will output a Transformer, which can transform data.
+
+            // In this example we define a CountFeatureSelectingEstimator, that selects slots in a feature vector that have more non-default 
+            // values than the specified count. This transformation can be used to remove columns with too many missing values.
+            // We also define a MutualInformationFeatureSelectingEstimator that selects the top k slots in a feature 
+            // vector based on highest mutual information between that slot and a specified label. Notice that it is possible to 
+            // specify the parameter `numBins', which controls the number of bins used in the approximation of the mutual information
+            // between features and label.
+            var pipeline = reader.MakeNewEstimator()
+                .Append(r =>(
+                    FeaturesCountSelect: r.Features.SelectFeaturesBasedOnCount(count: 695),
+                    Label: r.Label
+                    ))
+                .Append(r => (
+                    FeaturesCountSelect: r.FeaturesCountSelect,
+                    FeaturesMISelect: r.FeaturesCountSelect.SelectFeaturesBasedOnMutualInformation(r.Label, slotsInOutput: 5),
+                    Label: r.Label
+                    ));
+
+
+            // The pipeline can then be trained, using .Fit(), and the resulting transformer can be used to transform data. 
+            var transformedData = pipeline.Fit(data).Transform(data);
+
+            // Small helper to print the data inside a column, in the console. Only prints the first 10 rows.
+            Action<string, IEnumerable<VBuffer<float>>> printHelper = (columnName, column) =>
+            {
+                Console.WriteLine($"{columnName} column obtained post-transformation.");
+                int count = 0;
+                foreach (var row in column)
+                {
+                    foreach (var value in row.GetValues())
+                        Console.Write($"{value}\t");
+                    Console.WriteLine("");
+                    count++;
+                    if (count >= 10)
+                        break;
+                }
+
+                Console.WriteLine("===================================================");
+            };
+
+            // Print the data that results from the transformations.
+            var countSelectColumn = transformedData.AsDynamic.GetColumn<VBuffer<float>>(ml, "FeaturesCountSelect");
+            var MISelectColumn = transformedData.AsDynamic.GetColumn<VBuffer<float>>(ml, "FeaturesMISelect");
+            printHelper("FeaturesCountSelect", countSelectColumn);
+            printHelper("FeaturesMISelect", MISelectColumn);
+
+            // Below is the output of the this code. We see that some slots habe been dropped by the first trnsformation.
```

Comment:
>habe been dropped by the first trnsformation [](start = 76, length = 44)

typo

Reply:
Thanks for catching it!

---
In reply to: [235524636](https://github.com/dotnet/machinelearning/pull/1683#discussion_r235524636) [](ancestors = 235524636)